### ISSUE TITLE
I only have one 1080Ti GPU，Can I train a res18 network with the following command？How should I adjust the hyperparameters?

### ISSUE THREAD

**Issue by ashencode** [xingyizhou/CenterNet#393]
python main.py ctdet --exp_id coco_resdcn18 --arch resdcn_18 --batch_size 30--master_batch 4--lr 5e-4 --gpus 0   --num_workers 16

**Comment by qh332612340** [comment]
you can reduce the batch_size number
I also use 1080Ti GPU to train my own data, I set the batch_size number to 8

**Comment by HamsterHuey** [comment]
You should reduce the batch_size number and decrease the learning rate in a proportional manner as well. Smaller batch sizes should have smaller learning rates and the authors recommend a linear scaling based on this paper:
https://arxiv.org/abs/1706.02677
